common-pg/src/main/scala/com/socrata/pg/soql/SoQLAnalysisSqlizer.scala

```diff
@@ -96,10 +96,16 @@ object BinarySoQLAnalysisSqlizer extends Sqlizer[(BinaryTree[SoQLAnalysis[UserCo
     analyses match {
       case PipeQuery(l, Leaf(ra)) =>
         val (lpsql, _) = sql(l, tableNames, allColumnReps, reqRowCount, rep, typeRep, setParams, ctx, escape, fromTableName)
-        val chainedTableAlias = "x1"
-        val subTableName = "(%s) AS %s".format(lpsql.sql.head, chainedTableAlias)
-        val tableNamesSubTableNameReplace = tableNames + (TableName.PrimaryTable -> subTableName)
-        val primaryTableAlias = if (ra.joins.nonEmpty) Map((PrimaryTableAlias -> chainedTableAlias)) else Map.empty
+        val (primaryTableName, chainedTableAlias) = ra.from match {
+            case Some(tn@TableName(name, Some(alias))) if name == TableName.This =>
+              (tn.copy(alias = None), alias)
```

Comment:
Hm, so if the user tries to alias `this`, we ignore it?

Reply:
No, it is not ignored.  It is just the way we manage table aliases.